.github/workflows/release.yml

```diff
@@ -6,104 +6,136 @@ on:
       - "v*"
 
 jobs:
-
-  build-binary:
-    name: Build binary
+  wait-for-all-previous:
+    name: Wait for test and lint workflows
     runs-on: ubuntu-20.04
-    outputs:
-      VERSION: ${{ steps.set-version.outputs.VERSION }}
     steps:
-      - name: Set up Go 1.17
-        uses: actions/setup-go@v2
+      - name: Wait for all jobs finished on workflow Test and Lint
+        uses: lewagon/wait-on-check-action@v1.0.0
         with:
-          go-version: 1.17
-
-      - name: Get Go protoc compiler plugins
-        env:
-          GOLANG_PROTOBUF_VERSION: 1.3.5
-          GOGO_PROTOBUF_VERSION: 1.3.2
-          GRPC_GATEWAY_VERSION: 1.14.7
-        # Taken from: tendermintdev/sdk-proto-gen:v0.2
-        run: |
-          go get \
-            github.com/golang/protobuf/protoc-gen-go@v"$GOLANG_PROTOBUF_VERSION" \
-            github.com/gogo/protobuf/protoc-gen-gogo@v"$GOGO_PROTOBUF_VERSION" \
-            github.com/gogo/protobuf/protoc-gen-gogofast@v"$GOGO_PROTOBUF_VERSION" \
-            github.com/gogo/protobuf/protoc-gen-gogofaster@v"$GOGO_PROTOBUF_VERSION" \
-            github.com/grpc-ecosystem/grpc-gateway/protoc-gen-grpc-gateway@v"$GRPC_GATEWAY_VERSION" \
-            github.com/grpc-ecosystem/grpc-gateway/protoc-gen-swagger@v"$GRPC_GATEWAY_VERSION" \
-            github.com/regen-network/cosmos-proto/protoc-gen-gocosmos@latest
-
-      - name: Install buf
-        env:
-          PREFIX: "/usr/local"
-          VERSION: "1.0.0-rc8"
-        run: |
-          curl -sSL "https://github.com/bufbuild/buf/releases/download/v${VERSION}/buf-$(uname -s)-$(uname -m).tar.gz" | \
-          sudo tar -xvzf - -C "${PREFIX}" --strip-components 1
-
+          ref: ${{ github.sha }}
+          running-workflow-name: 'Wait for test and lint workflows'
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          wait-interval: 30
```

Comment:
Should we decrease the interval to 10 seconds?

Reply:
Firstly, it was 10 seconds by default but I increased it because of lags on starting for our workflows.